index.html

```diff
@@ -11,24 +11,20 @@
 <body>
     <div class= "header">
     <div class= "logo">
-   <div style="font-size: 34px; font-style: italic;" > <p><strong> World Swimming</strong></p> </div>
-       
-       
+   <div style="font-size: 34px; font-style: italic;" > <p><strong> World Swimming </strong></p> </div>
        
+              
        </div>
        
         <div class= "links"> 
            
         <a href="#1">Header link1</a>  
         <a href="#2">Header link2</a>
         <a href="#3">Header link3</a>
-       
-        
+               
         </div>
```

Comment:
Please consider using ul which can simplify and is a good practice for these types of items.